### DISCUSSION TITLE
pmdUpdate with SetTriggerMode 1

### DISCUSSION THREAD

**Discussion by VtyurinaSI** [ifm/ifm3d#466] [Q&A]
Hi! I got o3d353, and worked on the default TriggerMode, then changed it to 2. Everything was fine. But now I want to go back to free-run, but now in this mode I get the error PMD_TIMEOUT_ERROR = 1028 in response to pmdUpdate and PMD_UNKNOWN_HANDLE = 2049 in response to pmdGet3DCoordinates. Only the TriggerMode has changed. What could be the problem?

**Comment by lola-masson** [comment]
Hi @VtyurinaSI,

Can you confirm that the part number is O3D535? I am not able to find this part number in the system, could it be a different number?
Can you also provide the firmware version for the Device?

The commands you mention belong to the pmd sdk, I believe, which is no longer supported and has been replaced with the ifm3d API. It might make sense to transition to this API for better support.

**Reply by VtyurinaSI** [reply]
Sorry! The part number is O3D353. 
![20241003_094954](https://github.com/user-attachments/assets/e8683971-fe10-4c6a-9fb9-e167f75b0f82)
![Uploading 20241003_094954.jpg…]()

**Reply by lola-masson** [reply]
Thank you. This is a model that was obsoleted in 2018, and it seems that you are using the pmd sdk, which is also deprecated and was replaced with the ifm3d API. I think the best way forward would be to arrange a call directly with you to see how we can troubleshoot the issue. Could you get in touch with your ifm sales representative, or alternatively email support.efector.object-ident@ifm.com so we can communicate with you directly. Thanks!

**Reply by VtyurinaSI** [reply]
I sent a message to support.efector.object-ident@ifm.com but have not received a response yet
